src/components/dashboard/styles/index.css

```diff
@@ -331,4 +331,21 @@ nav {
 }
 .display-name{
   font-size: 20px;
-}
\ No newline at end of file
+}
+#dictionary {
+  min-height: 8rem;
+}
+
+#dictionaryHeader{
+  background: #2b6d27;
+}
+
+#dictionary-owner {
+  background: #225a21;
+}
+
+#viewDetails{
+  margin-bottom: 6%;
+  font-size: 11px;
+  border: none;
+}
```

Comment:
There is a dictionary css file, i think it might make more sense if all the css for dictioanries is in that file rather than in the index file of the dashboard.

Reply:
True

Reply:
The reason I placed that there, is I am reusing the CSS that Emma had written hence avoid repetition of code. In the dictionary CSS file